Makefile

```diff
@@ -23,24 +27,28 @@ test-iPad:
 		xcodebuild \
 		-project Hammer.xcodeproj \
 		-scheme Hammer \
-		-destination "name=iPad Pro (12.9-inch) (4th generation)" \
-		test
+		-destination "name=iPad Pro (12.9-inch) (6th generation)" \
+		test \
+		$(NO_CODE_SIGN_SETTINGS)
 
 test-iPhone:
 	set -o pipefail && \
 		xcodebuild \
 		-project Hammer.xcodeproj \
 		-scheme Hammer \
-		-destination "name=iPhone 11" \
-		test
+		-destination "name=iPhone 15" \
+		test \
+		$(NO_CODE_SIGN_SETTINGS)
 
-test-iPhone-iOS12:
+test-iPhone-iOS15:
 	set -o pipefail && \
 		xcodebuild \
 		-project Hammer.xcodeproj \
 		-scheme Hammer \
-		-destination "name=iPhone 6" \
-		test
+		-destination "name=iPhone 11" \
+		-sdk iphonesimulator15.0 \
```

Comment:
This a bit unfortunate but GitHub Actions deprecated the `macos_11` runner which would let us test all the way to iOS 13.7. We can figure out alternatives if this is critical but we'd need to download and install a large runtime file.

Reply:
I think this is fine